projects/components/src/info-icon/info-icon.component.ts

```diff
@@ -0,0 +1,28 @@
+import { ChangeDetectionStrategy, Component, Input, TemplateRef } from '@angular/core';
+import { IconType } from '@hypertrace/assets-library';
+import { IconSize } from '../icon/icon-size';
+
+@Component({
+  selector: 'ht-info-icon',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <ng-container *ngIf="this.info"
```

Comment:
Formatting seems weird 🤔

Reply:
I have used prettier, is there anything specific?